python/graphstorm/gconstruct/remap_result.py

```diff
@@ -172,6 +174,120 @@ def _get_file_range(num_files, rank, world_size):
 
     return start, end
 
+def remap_node_emb(emb_ntypes, node_emb_dir,
```

Comment:
how much is this different from the one below?

Reply:
basically, the input file names and the output table column names are different.

Reply:
if it's just the difference of file names, do we really need two functions that implement them separately?